src/Configs/Constants.vala

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2017 Alonso Enrique <alons45@gmail.com> (http://github.com/alons45/quotes)
+
+* This program is free software: you can redistribute it
+* and/or modify it under the terms of the GNU General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be
+* useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+* Public License for more details.
+*
+* You should have received a copy of the GNU General Public License along
+* with this program. If not, see http://www.gnu.org/licenses/.
+*/
+
+namespace Quotes.Configs {
+
+    /**
+     * The {@code Constants} class is responsible for defining all
+     * the constants used in the application.
+     *
+     * @since 0.1.0
+     */
+    public class Constants {
+
+        public abstract const string PROGRAME_NAME = (_("Quotes"));
+        public abstract const string ID = "com.github.alonsoenrique.quotes";
```

Comment:
As far as I searched this constant is only used at line 32 of `Application.vala`. Also, the above constant `PROGRAME_NAME` does not seem to be used in any code. So I think you can hardcode `com.github.alonsoenrique.quotes` and remove this file itself.